pandas/core/tools/datetimes.py

```diff
@@ -1011,7 +993,7 @@ def to_datetime(
       are constant:
 
     >>> from datetime import datetime
-    >>> pd.to_datetime(["2020-01-01 01:00 -01:00", datetime(2020, 1, 1, 3, 0)])
+    >>> pd.to_datetime(["2020-01-01 01:00:00-01:00", datetime(2020, 1, 1, 3, 0)])
     DatetimeIndex(['2020-01-01 01:00:00-01:00', '2020-01-01 02:00:00-01:00'],
                   dtype='datetime64[ns, pytz.FixedOffset(-60)]', freq=None)
```

Comment:
I made this change because of https://github.com/pandas-dev/pandas/issues/50025

Else, I could've changed the return type to be `Index(..., dtype=object)` - either way, I think we should resolve that separately